src/index_emails.py

```diff
@@ -123,10 +123,11 @@ def normalize_email(email_in):
 
 def content_type_is_ignored(content_type):
     if tornado.options.options.index_bodies_ignore_content_types:
+        content_type_lcase = content_type.lower()
         ignore_types = tornado.options.options.index_bodies_ignore_content_types.split(",")
         for to_ignore in ignore_types:
             if to_ignore:
-                if to_ignore.strip().lower() in content_type.lower():
+                if to_ignore.strip().lower() in content_type_lcase:
```

Comment:
Using `content_type_lcase` is a good start but what I really meant is `to_ignore.strip().lower()` should be done at startup time of the program, not on every invocation of this function.